File: app/src/main/java/cordova/plugin/photo/editor/photoeditor/FontManager.java
package cordova.plugin.photo.editor.photoeditor;

import android.content.Context;
import android.graphics.Typeface;

public class FontManager {

    public static final String FONTAWESOME ="FAsolid.ttf";
    public static final String EMOJIONE = "emojione-android.ttf";

    public static Typeface getTypeface(Context context, String font) {
        return Typeface.createFromAsset(context.getAssets(), font);
    }
}

File: app/src/main/java/cordova/plugin/photo/editor/photoeditor/FontManager.kt
package cordova.plugin.photo.editor.photoeditor

import android.content.Context
import android.graphics.Typeface


object FontManager {

    val FONTAWESOME = "FAsolid.ttf"
    val EMOJIONE = "emojione-android.ttf"

    fun getTypeface(context: Context, font: String): Typeface {
        return Typeface.createFromAsset(context.getAssets(), font)
    }
}